lib/repositories/stubs/stub_account_repository.dart

```diff
@@ -1,141 +1,38 @@
 import 'package:pispapp/models/account.dart';
+import 'package:pispapp/models/fsp_info.dart';
+import 'package:pispapp/models/party_id_info.dart';
 import 'package:pispapp/repositories/interfaces/i_account_repository.dart';
 
-List<Account> getMyDummyAccounts() {
-  final List<Account> listOfAccounts = <Account>[];
-
-  listOfAccounts.add(Account(
+final List<Account> accounts = <Account>[
+  Account(
     alias: 'Personal',
-    phoneNumber: 'IN1233323987',
-    name: 'John Doe',
-    balance: '100000',
-    accountNumber: '120980988878828',
-    bankName: 'Bank of India',
-    linked: true,
-  ));
-  listOfAccounts.add(Account(
+    partyInfo: PartyIdInfo(
+      fspId: 'DJCICFQ1919',
+      partyIdentifier: 'IN1233323987',
+    ),
+    fspInfo: FspInfo(fspId: 'DJCICFQ1919', fspName: 'Bank of India'),
+    consentId: '555',
+    userId: 'vXiSsQglsFYXqVkOHNKKFhnuAAI2',
```

Comment:
I think you shouldn't hardcode your own `userId` here since people will use their own account. Instead, I think the `StubAccountRepository` should do any filtering based on `userId`.

Reply:
Removed all filtering